File: solutions/java/find-maximum.java
public class FindMaximum {
    public static int findMax(int[] arr) {
        int max = arr[0];
        for (int n : arr) if (n > max) max = n;
        return max;
    }
    public static void main(String[] args) {
        System.out.println(findMax(new int[]{1,5,3}));
    }
}

File: solutions/cpp/find-maximum.cpp
#include <iostream>
using namespace std;

int findMax(int arr[], int n) {
    int maxv = arr[0];
    for (int i = 1; i < n; i++)
        if (arr[i] > maxv) maxv = arr[i];
    return maxv;
}

int main() {
    int a[3] = {1, 5, 3};
    cout << findMax(a, 3);
}
